spec/acceptance/basic_spec.rb

```diff
@@ -28,4 +28,10 @@
   describe port(80) do
     it { is_expected.to be_listening }
   end
+
+  describe command("curl -s http://centos7.mercury.example.com/pulp/api/v3/status/ -w '%{response_code}' -o /dev/null") do
```

Comment:
You can use `localhost` instead of the hostname (which will differ for every system). The alternative is to use `fact('fqdn')` but for now `localhost` should do as well.